packages/vscode-extension/WHATISNEW.md

```diff
@@ -6,6 +7,18 @@
 - [2.8.0 - Oct 18 2021](#280---oct-18-2021)
 - [2.7.0 - Sep 17 2021](#270---sep-17-2021)
 
+## 3.1.2 - Jan 10, 2022
+Incremental version for Teams Toolkit with multiple bugs fixes and the following updates:
+
+New Features:
+- Use Service Principle to login Azure account in CICD template.
```

Comment:
How about adding `remove userdata from cicd template` in enhancement section? refer to this completed feature: https://dev.azure.com/msazure/Microsoft%20Teams%20Extensibility/_workitems/edit/12718019

Reply:
Agree, we have this enhancement for CICD as well

Reply:
I have a little concern to mention remove user data from cicd template, user may think we previously reveal user data and are not secure for user who already used CICD template in past time period.

Reply:
What about "CICD don't require checkin user.data file anymore"?

Reply:
How about "Improve CICD template by reducing dependency on TeamsFx metadata file"?

Reply:
LGTM